includes/data-port/class-sensei-data-port-job.php

```diff
@@ -0,0 +1,341 @@
+<?php
+/**
+ * File containing the Sensei_Data_Port_Job class.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Superclass of data import/export jobs. It provides basic functionality like logging, maintaining state, cleanup and
+ * running data port tasks which are registered by subclasses.
+ */
+abstract class Sensei_Data_Port_Job implements Sensei_Background_Job_Interface, JsonSerializable {
```

Comment:
I'm not sure this name makes sense. I don't consider "Port" to represent commonality between "Import" and "Export". To me "Port" represents something different.

Although it's more verbose, to avoid confusion I wonder if we should call this (and other related classes) something like `Sensei_Data_Import_Export_Job`?

Reply:
Since I had trouble finding a proper wording, I looked it up and come up with this name from here:
https://english.stackexchange.com/questions/167620/what-is-a-single-word-that-means-both-import-and-export
https://ux.stackexchange.com/questions/106584/another-way-for-saying-importing-and-exporting

There might be a bit confusion with an actual 'port' so this is the reason I named it 'data port' instead. In general I would be against of using the Import and Export because it names the superclass from the names of the subclasses and fails to communicate that this is the common functionality.

What I mean is that we can say that the 'Import' class is a 'Data Port' class however it would make no sense to say that the 'Import' class is an 'Import Export' class. We could use an other name e.g. 'Transfer'?

Reply:
Interesting. I'm not a huge fan because of the ambiguity, but I don't have a compelling alternative. Naming is hard 😅